policy/score_calculator_test.go

```diff
@@ -239,7 +239,24 @@ func TestBandedScores(t *testing.T) {
 				// 10 high checks
 				{Value: &explorer.ImpactValue{Value: 80}},
 			},
-			out: &Score{Value: 45, ScoreCompletion: 100, DataCompletion: 66, Weight: 20, Type: ScoreType_Result},
+			out: &Score{Value: 1, ScoreCompletion: 100, DataCompletion: 66, Weight: 20, Type: ScoreType_Result},
+		},
+		{
+			in: []*Score{
+				// 10 critical checks (9ok, 1not)
+				{Value: 100, ScoreCompletion: 100, DataCompletion: 80, DataTotal: 5, Weight: 1, Type: ScoreType_Result},
+				{Value: 100, ScoreCompletion: 100, DataCompletion: 100, DataTotal: 1, Weight: 9, Type: ScoreType_Result},
+				// 10 high checks (ok)
+				{Value: 100, ScoreCompletion: 100, DataCompletion: 33, DataTotal: 3, Weight: 10, Type: ScoreType_Result},
+			},
+			impacts: []*explorer.Impact{
+				// 10 critical checks
+				{Value: &explorer.ImpactValue{Value: 100}},
+				{Value: &explorer.ImpactValue{Value: 100}},
+				// 10 high checks
+				{Value: &explorer.ImpactValue{Value: 80}},
+			},
+			out: &Score{Value: 100, ScoreCompletion: 100, DataCompletion: 66, Weight: 20, Type: ScoreType_Result},
```

Comment:
I added a case with all passing checks. I would expect if everything passed, then the policy score should also be 100. In the original code that is not the case. It works now, however, the values for the other cases changed too...